### PR TITLE
[WIP] Bump conda reqs to bump toolchain

### PR DESCRIPTION
**Pull request by jerryz123**
**Related PRs / Issues**:

**Type of change**:
- [ ] Bug fix
- [ ] New feature
- [ ] Other enhancement

**Impact**:
- [ ] RTL change
- [ ] Software change (RISC-V software)
- [ ] Build system change
- [ ] Other

**Contributor Checklist**:
- [ ] Did you set `main` as the base branch?
- [ ] Is this PR's title suitable for inclusion in the changelog and have you added a `changelog:<topic>` label?
- [ ] Did you state the type-of-change/impact?
- [ ] Did you delete any extraneous prints/debugging code?
- [ ] Did you mark the PR with a `changelog:` label?
- [ ] (If applicable) Did you update the conda `.conda-lock.yml` file if you updated the conda requirements file?
- [ ] (If applicable) Did you add documentation for the feature?
- [ ] (If applicable) Did you add a test demonstrating the PR?

- [ ] (If applicable) Did you mark the PR as `Please Backport`?

### CODE REVIEW

File: conda-reqs/chipyard.yaml [hunk 1]
```diff
@@ -46,10 +46,10 @@ dependencies:
     - findutils
     - rsync
     - psutil
-    - doit=0.35.0
+    - doit
     - gitpython
     - humanfriendly
-    - e2fsprogs
+    # - e2fsprogs
```
Comment (abejgonzalez) [inline conda-reqs/chipyard.yaml:52]: Can you build buildroot without this?

File: conda-reqs/chipyard.yaml [hunk 2]
```diff
@@ -46,10 +46,10 @@ dependencies:
     - findutils
     - rsync
     - psutil
-    - doit=0.35.0
+    - doit
```
Comment (abejgonzalez) [inline conda-reqs/chipyard.yaml:49]: This needs to be greater than at minimum. v34... doesn't work with the most recent marshal

File: conda-reqs/chipyard.yaml (hunk 1 above)
Comment (jerryz123) [reply conda-reqs/chipyard.yaml:52]: I don't think so. Will need to think of a solution once I fix the other issues with this PR.

File: conda-reqs/chipyard.yaml (hunk 2 above)
Comment (jerryz123) [reply conda-reqs/chipyard.yaml:49]: I'll add >= 0.35 here

**Pull request closed without merging**